### DISCUSSION TITLE
Ego in f1 22

### DISCUSSION THREAD

**Discussion by Pstrykos12** [Ryder25/Ego-Engine-Modding#69] [Q&A] [answered]
Hello, I write about the ego erp archiver to f1 22 I would like to know when you can count and will you make a version for this game?

**Comment by ptasev** [comment, marked as answer]
I took a look some time ago, but couldn't find any obvious issue.  I don't have time to look at it any time soon.

**Reply by Pstrykos12** [reply]
and it is impossible to somehow determine why after replacing files it crashes?

**Reply by ptasev** [reply]
Members of the discord community helped investigate the problem and found it was an easy fix. A new version has been released with support.
